timemachine/cpp/src/neighborlist.cu

```diff
@@ -87,7 +87,7 @@ void Neighborlist<RealType>::compute_block_bounds_host(
         d_box,
         static_cast<cudaStream_t>(0)
     );
-
+    // Does this need to peek at the last error?
     cudaDeviceSynchronize();
```

Comment:
Not sure - I don't think cudaDeviceSynchronize() can ever fail but might be good to double check.